ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -104,6 +105,22 @@ def _resolve_service_class(service_class: str, display_name: str) -> str:
     "--directory-out",
     help="Output directory for measurement files. Default: '<current_directory>/<display_name>'",
 )
+@click.option(
+    "-a",
+    "--annotations-description",
+    help="Short description for the Measurement",
+)
+@click.option(
+    "-c",
+    "--collection",
+    help="The collection that this measurement belongs to.",
```

Comment:
Can we add something like this to clarify (text from the .proto file)?
//   - Expected format: "." delimited namespace/hierarchy case-insensitive string
//   - Example: "CurrentTests.Inrush"

Reply:
Yes, I was thinking to add it to this help.
help="The collection that this measurement belongs to."